src/live_restore/live_restore_server.c

```diff
@@ -152,16 +170,14 @@ __live_restore_worker_run(WT_SESSION_IMPL *session, WT_THREAD *ctx)
     /* FIXME-WT-13897 Replace this with an API call into the block manager. */
     WT_FILE_HANDLE *fh = bm->block->fh->handle;
 
-    __wt_verbose_debug3(
-      session, WT_VERB_FILEOPS, "Live restore worker: Filling holes in %s", work_item->uri);
+    __wt_verbose_debug1(
+      session, WT_VERB_LIVE_RESTORE, "Live restore worker: Filling holes in %s", work_item->uri);
     ret = __wti_live_restore_fs_fill_holes(fh, wt_session);
-    __wt_verbose_debug3(session, WT_VERB_FILEOPS,
+    __wt_verbose_debug1(session, WT_VERB_LIVE_RESTORE,
       "Live restore worker: Finished finished filling holes in %s", work_item->uri);
 
     /* Free the work item. */
     __live_restore_free_work_item(session, &work_item);
-    WT_STAT_CONN_SET(
-      session, live_restore_queue_length, __wt_atomic_sub64(&server->work_items_remaining, 1));
```

Comment:
This was moved up to where we actually remove the item from the queue.

Reply:
Yes the reason I put it here is that if you were to view the FTDC you'd see 0 items remaining but the live restore would not be complete. It was a UX based decision rather than explaining why the queue length is zero but it's still running.

Reply:
Should we rename the stat to `work_remaining`? `queue_length` is misleading for what we want to report to the user.

Reply:
I renamed the stat.